staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -131,6 +131,13 @@ type Config struct {
 	// Version forces a specific version to be used (if registered)
 	// Do we need this?
 	// Version string
+
+	// ProxyURL is the URL to the proxy to be used for all requests made by this
+	// client. URLs with "http", "https", and "socks5" schemes are supported.
```

Comment:
socks5 currently won't work for exec/attach/log streaming commands. Have you confirmed that these work over http/https proxy?

Reply:
as of today, we have to create custom alias like:
`kubeclt` => default binary / no proxy
`kdev` => set `http_proxy` for dev / forward to `kubectl` binary / unset `http_proxy` (not to polute the shell)
`kprd` ....

That's kind of a mess but `attach/exec/log` seems to work